packages/utils/src/object.ts

```diff
@@ -274,3 +315,41 @@ export function normalize(input: any, depth?: number): any {
     return '**non-serializable**';
   }
 }
+
+/** Merges provided array of keys into */
```

Comment:
Even though I get what this function is doing, I'm not sure what this comment means. (Sorry, I know this is a WIP.)

Reply:
Reworded to:

```
/**
 * Given any captured exception, extract it's keys and create a sorted
 * and truncated list that will be used inside the event message.
 * eg. `Non-error exception captured with keys: foo, bar, baz`
 */
```